### PR TITLE
add FillReferenceInfo for udp

### PR DESCRIPTION
**Pull request by superyyrrzz**

### CODE REVIEW

**Comment by vwxyzh** [comment]
:shipit:

**Comment by vicancy** [comment]
can we reuse the one in MREF?

**Comment by superyyrrzz** [comment]
Seems hard as they have different PageViewModel

---
In reply to: [299804804](https://github.com/dotnet/docfx/pull/1641#issuecomment-299804804) [](ancestors = 299804804)

**Comment by hellosnow** [comment]
>             // TODO: inheritance tree is removed from data, wait for the design decision

minor: please help to update inheritance tree to the yml back, thanks

---
Refers to: test/Microsoft.DocAsCode.Build.UniversalReference.Tests/UniversalReferenceDocumentProcessorTest.cs:49 in c5f38c6. [](commit_id = c5f38c680db855bebb539947fdf8d7ee6afbcadd, deletion_comment = False)

File: plugins/Microsoft.DocAsCode.Build.UniversalReference/ModelConverter/ModelConverter.cs [hunk 1]
```diff
@@ -134,6 +136,7 @@ public static List<ApiLanguageValuePair<List<ApiExceptionInfoBuildOutput>>> ToAp
             {
                 return null;
             }
+            values = values ?? new SortedList<string, List<ExceptionInfo>>();
```
Comment (hellosnow) [inline plugins/Microsoft.DocAsCode.Build.UniversalReference/ModelConverter/ModelConverter.cs]: >values [](start = 12, length = 6)

judge in ToApiListInDevLangs method whether the values is null ?

**Comment by hellosnow** [comment]
:shipit:

**Comment by superyyrrzz** [comment]
>             // TODO: inheritance tree is removed from data, wait for the design decision

OK. will remove this in next PR.

---
In reply to: [300048065](https://github.com/dotnet/docfx/pull/1641#issuecomment-300048065) [](ancestors = 300048065)

---
Refers to: test/Microsoft.DocAsCode.Build.UniversalReference.Tests/UniversalReferenceDocumentProcessorTest.cs:49 in c5f38c6. [](commit_id = c5f38c680db855bebb539947fdf8d7ee6afbcadd, deletion_comment = False)

**Pull request merged**